--- aseba/aseba/vm/natives.h	2022-04-08 08:50:26.000000000 +0200
+++ aseba/aseba/vm/natives_win.h	2022-04-08 08:50:36.000000000 +0200
@@ -29,7 +29,7 @@
 
 #ifdef _MSC_VER
 #pragma warning( disable: 4200)	// disable 0 size arrays warning
-#define inline __inline			// msvc does not support the standart C99/C++ static inline construct
+//#define inline __inline			// msvc does not support the standart C99/C++ static inline construct
 #endif
 
 /**
